src/components/dialogs/field-hook.js

```diff
@@ -78,15 +78,45 @@ export const useTextValue = ({
     return [value, field];
 };
 
-export const useFileValue = ({ active }) => {
+export const useFileValue = ({ active, fileExceedsLimitMessage }) => {
     const selectedFile = useSelector((state) => state.selectedFile);
+    const intl = useIntl();
     const dispatch = useDispatch();
+    const [isFileOk, setIsFileOk] = useState(false);
+    const [fileError, setFileError] = useState();
 
     const field = <UploadCase />;
     useEffect(() => {
         dispatch(removeSelectedFile());
```

Comment:
It's not he role of this custom hook to set the redux state.selectedFile but it manages its reset in this useEffect. Seems strange behavior for me.

Reply:
will be fixed later